src/efjm.tw2

```diff
@@ -22,7 +23,11 @@ Escape from Juggalo Mountain
 [[Jump to the junk heap|RM-junkheap-entrance-nameprompt]]
 [[Jump to The Church - First visit|The Church - First visit - Name prompt]]
 [[Jump to The Church - Rap battle|The Church - Rap battle name prompt]]
-(set: $justWokeUp to true)
+{
+  (set: $justWokeUp to true)
+  (set: $dissolve to (transition: 'dissolve') + (transition-time: 3s))
+  (set: $monospace to (css: 'font-family: "Courier New", "Courier", monospace'))
```

Comment:
Any chance we can get anything more future-ish bullshit here (a la Blade Runner)? I don't hate Courier, but would love to have this look like old computer display text.

Something like this: https://fonts.google.com/specimen/VT323